### DISCUSSION TITLE
Release 0.47.4 is now available

### DISCUSSION THREAD

**Discussion by Haishi2016** [eclipse-symphony/symphony#120] [Announcements]
February 2024 release (0.47.4) is [here](https://github.com/eclipse-symphony/symphony/releases/tag/0.47.4)!

We focused on code quality and test coverage improvements in this release. 

## What's Changed
* gated deployment with script doc fix by @Haishi2016 in https://github.com/eclipse-symphony/symphony/pull/94
* Enable local MQTT and docker test in Go pipeline by @LeiQL in https://github.com/eclipse-symphony/symphony/pull/86
* Guard invalid null request ID in http and k8s state providers by @LeiQL in https://github.com/eclipse-symphony/symphony/pull/87
* Improve test coverages, add logs and fix bugs in symphony-api by @msftcoderdjw in https://github.com/eclipse-symphony/symphony/pull/102
* Improve logging by @LeiQL in https://github.com/eclipse-symphony/symphony/pull/89
* fix regression in updating {VERSION} in values.yaml by @Haishi2016 in https://github.com/eclipse-symphony/symphony/pull/112

**Full Changelog**: https://github.com/eclipse-symphony/symphony/compare/0.47.3...0.47.4

## Get the latest version

* Download latest maestro
   `wget -q https://raw.githubusercontent.com/eclipse-symphony/symphony/master/cli/install/install.sh -O - | /bin/bash`
* Use **maestro up** to install/update Symphony

**Comment by Pellifer** [comment]
Good job team! I will try out this new release.
